backend/src/common/util/workflow.go

```diff
@@ -135,6 +135,14 @@ func (w *Workflow) ScheduledAtInSecOr0() int64 {
 	return 0
 }
 
+func (w *Workflow) FinishedAt() int64 {
+	if w.Status.FinishedAt.IsZero(){
```

Comment:
this doesn't need to be nil-checked?

Reply:
no finishedAt is a struct not a pointer so it won't be nil. checking zero should be good.